mne/viz/_3d.py

```diff
@@ -448,6 +448,11 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         * brain: one of 'pial', 'white', 'inflated', or 'brain'
           (alias for 'pial').
 
+        Can be dict to specify alpha values for each surface. Use None
+        to specify default value. Specified values must be between 0 and 1.
+        Example:
+        ``surfaces={'brain': 0.4, 'outer_skull': 0.6, 'head': None}``
+
         Defaults to 'auto', which will look for a head surface and plot
```

Comment:
you can get nicer rendering with a code block:
```suggestion
        to specify default value. Specified values must be between 0 and 1,
        for example::

            surfaces=dict(brain=0.4, outer_skull=0.6, head=None)

        Defaults to 'auto', which will look for a head surface and plot
```

Reply:
Great! will incorporate